.github/workflows/gitleaks.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       REPO: https://github.com/CMSgov/ab2d
-      REMOTE_EXCLUDES_URL: https://raw.githubusercontent.com/semanticbits/ab2d-gitleaks-automation/master/ab2d/gitleaks.toml
+      REMOTE_EXCLUDES_URL: https://raw.githubusercontent.com/gitleaks/gitleaks/master/config/gitleaks.toml
```

Comment:
We shouldn't need to reference the default config, right? Also, could we upgrade to latest?

Reply:
so, i'll have to add an extra unpack step in the script